cn.apipost.apipost.metainfo.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id type="desktop">cn.apipost.apipost</id>
+  <name>apipost</name>
+  <summary>Apipost = Postman + Swagger + Mock + Jmeter, Backend, frontend, testing, and online collaboration with real-time content synchronization</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://wiki.apipost.cn/docs/introduction</project_license>
```

Comment:
is this redistributable? If not use extra-data

Reply:
not use extra-data, because the downloaded files are unstable, old version file will expire and be cleared

Reply:
If it's not redistributable, you don't have a choice.

Reply:
changed this https://github.com/jcleng/flathub/commit/306d3cf08de568d80e7a725085304a969146f4b9#diff-5730d8d5f1a627350fa7fe51ec6a7ea3d664823ea5059d93c0232d2fec150b98R115